codec-redis/src/main/java/io/netty/handler/codec/redis/DefaultBulkStringRedisContent.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.DefaultByteBufHolder;
+import io.netty.util.internal.StringUtil;
+
+/**
+ * A default implementation of {@link BulkStringRedisContent}.
+ */
+public class DefaultBulkStringRedisContent extends DefaultByteBufHolder implements BulkStringRedisContent {
```

Comment:
@jongyeol - In general we try to keep access as "strict as possible". This may make maintenance and changes easier down the line. If they are intended to be overridden that is fine, but if there isn't a strong use case we can always open up access later.

@normanmaurer - wdyt?